rr_computer_vision/src/traffic_light.cpp

```diff
@@ -1,19 +1,156 @@
-/*
- * @file rr_traffic_light
- * @author Adrian Malaran(AdrianMalaran)
- * @author Toni Ogunmade(oluwatoni)
- * @author Jamie Kim
- * @author Jason Leung
- * @competition IARRC 2018
- */
+  //  @file traffic light detection
+  //  @author Yuchi(Allan) Zhao
+  //  @competition IARRC 2019
 
 #include "traffic_light.hpp"
 
-/**@name TrafficLightDetector
- * @brief Constructor
- */
-TrafficLight::TrafficLight() {
+TrafficLightProcessor::TrafficLightProcessor(ros::NodeHandle nh) : it_(nh_)  {
+    red_light_detected = false;
```

Comment:
Its cleaner to initialize all these member variables the same way you did the image transport object, like so: 
`TrafficLightProcessor::TrafficLightProcessor(ros::NodeHandle nh) : 
    it_(nh_), 
    red_light_detected(false),
    red_Pixel_Counter(0) {`